Kbs.Business/BoatType/BoatTypeValidator.cs

```diff
@@ -31,4 +31,17 @@ public Dictionary<string, string> ValidateForCreate(BoatTypeEntity boatType)
 
         return validationResult;
     }
+    
+    public Dictionary<string, string> ValidatorForUpdate(BoatTypeEntity boatType)
+    {
+        ThrowHelper.ThrowIfNull(boatType);
+        Dictionary<string, string> validationResult = new();
+
+        if (boatType.Speed <= 0)
```

Comment:
Je zou nog wat extensiever kunnen testen, ook met of stoelen defined is bijv.

Reply:
de waarden van de stoelen staat by default op de waarde die er in de database staat. Deze kan dan toch niet undefined zijn?

Reply:
Op het moment staan er waarden in de database die niet geldig zijn. Een aantal boottypen hebben als Seats het getal 0 staan, of hoger dan 8. Dit kon destijds omdat de enum anders / niet defined was. Dit kan later natuurlijk nog steeds voorkomen (veranderingen in datastructuur). Daarom denk ik dat het wel handig is om extensief te controleren.

Daarnaast is het wel handig als de naam gecontroleerd wordt.